website/docs/node-operators/infrastructure/seed-nodes.mdx

```diff
@@ -91,6 +91,35 @@ Infrastructure maintenance and updates are communicated through:
 - Project documentation updates
 - Community announcements
 
+## Deployment (o1Labs Employees Only)
+
+<!-- prettier-ignore-start -->
+
+:::caution o1Labs Employees Only
+
+This section is for o1Labs employees only. If you do not have access to the
+required repositories, contact the platform lead.
+
+:::
+
+<!-- prettier-ignore-stop -->
+
+o1Labs uses [ArgoCD](https://argo-cd.readthedocs.io/) to maintain the
+infrastructure through GitOps.
+
+To update the seed nodes infrastructure:
+
+1. Access the private repository:
+   `https://github.com/o1-labs/gitops-infrastructure`
+2. Create a new branch for your changes
+3. Edit: `applications/auto-deployed/mina-rust-standard-seeds.yaml`
```

Comment:
Both files plain-nodes and seed-nodes point to the same yaml?

Reply:
Yes.